build.gradle

```diff
@@ -527,7 +527,9 @@ integTest {
             }
         }
     }
-
+    filter {
```

Comment:
Why is this change required here?

Reply:
The integTests were failing for few of them and i could find the below PR which involves the exclusion of this test case in integTestRemote Task but not in integTest.

https://github.com/opensearch-project/cross-cluster-replication/pull/1379

I have for now excluded this particular test case from the integTest to proceed for now. Let me know if any other plan to fix this test case as I can remove them from my current PR?

Reply:
the test was disabled for remoteIntegTest due to a different issue. 
@astute-decipher did we cleanup the setting?

Reply:
@ankitkala Shall I remove that filter then?

Reply:
yes

Reply:
done